src/app/base/components/node/OverviewCard/MachineStatusCard/MachineStatusCard.tsx

```diff
@@ -74,6 +74,9 @@ const MachineStatusCard = ({ machine }: Props): JSX.Element => {
             </i>
           )}
           {machine.status}
+          {isEphemerallyDeployed(machine) && (
```

Comment:
In theory, `ephemerally_deployed` could only be true if the machine also has a "deployed" status, but just to be sure I'd still add a check here just in case.

```suggestion
          {machine.status === NodeStatus.DEPLOYED && isEphemerallyDeployed(machine) && (
```

Reply:
I think this check is already met on the utility function like so:

```javascript
export const isEphemerallyDeployed = (machine: Machine | null) => {
  return (
    machine &&
    machine.status_code === NodeStatusCode.DEPLOYED &&
    machine.ephemeral_deploy
  );
};
```
I could add this (`machine.status === NodeStatus.DEPLOYED`) also to the check to make it more thorough, what do you think?

Reply:
Oh, if it's already in the util function then that's fine IMO, please disregard!